samples/data-grid/basic/column-options/demo.css

```diff
@@ -0,0 +1,15 @@
+@import url("https://code.highcharts.com/datagrid/css/datagrid.css");
```

Comment:
This might be my mistake but we should use `dashbaords/css/datagrid.css` in the public demo. The only exceptions are the cypress demos which now also should work with `dasboards...` I presume 🤔